File: 002/002.go
package main

import (
	"fmt"
	"sort"
)

func main() {
	var str string
	asciiCount := map[string]int{}

	fmt.Scan(&str)

	for _, c := range str {
		asciiCount[string(c)] += 1
	}

	var ks []string
	for k := range asciiCount {
		ks = append(ks, k)
	}

	sort.Strings(ks)

	for _, k := range ks {
		fmt.Println(k, " => ", asciiCount[k])
	}
}

File: 002/002.js
let asciiCount = [];
const input = require("fs").readFileSync("/dev/stdin", "utf8").trim().split('');

for (const index in input) {
    const target = asciiCount.find((ascii) => ascii.char === input[index])

    if (target) {
        target.count = target.count + 1
    } else {
        asciiCount.push({char: input[index], count: 1})
    }
}

asciiCount.sort((s, t) => (s.char < t.char ? -1 : 1))
asciiCount.forEach((elem, _) => {
    console.log(`${elem.char} => ${elem.count}`)
})
